src/codegeneration/ModuleTransformer.js

```diff
@@ -113,7 +110,7 @@ export class ModuleTransformer extends TempVarTransformer {
 
   wrapModule(statements) {
     var functionExpression;
-    if (transformOptions.require) {
+    if (this.options_.transformView().require) {
```

Comment:
Up to you. I was mostly thinking about reducing the code duplication. It should be safe to assume that the options do not change during transformation.